utilities/metadata/project.schema.json

```diff
@@ -141,6 +141,48 @@
         "description"
       ]
     },
+    "dependency": {
```

Comment:
we now have `dependency`, `application_dependencies`, `dependencies`, and `depends`. I think we need to improve naming and make sure we need all of these

Reply:
thanks, I missed some changes, now the only new is `depends` which corresponds to `--with operator1;operator2` in the CLI commands